testsuite/features/reposync/allcli_update_activationkeys.feature

```diff
@@ -205,6 +205,26 @@ Feature: Update activation keys
     And I click on "Update Activation Key"
     Then I should see a "Activation key Build host Key x86_64 has been modified" text
 
+@uyuni
```

Comment:
buildhost is sle15sp4 in the uyuni CI jenkins because it needs to be the same OS that builds the images

Reply:
fixed